crux-mir/translate_libs.sh

```diff
@@ -2,15 +2,24 @@
 
 set -e
 
+TARGET=${1:-$(rustc -vV | sed -n 's/host: //p')}
```

Comment:
It would be helpful to add some comments describing what these various commands are doing. For instance, I had to stare at this a while (and Google what `rustc -vV`'s output looks like) before I realized that it was scraping the host target triple from `rustc --version --verbose`'s output.

Reply:
Heh, I didn't know that `-vV` could be written as `--version --verbose` either. I got it from [here](https://stackoverflow.com/questions/52996949/how-can-i-find-the-current-rust-compilers-default-llvm-target-triple) which got it from [cargo's source code](https://github.com/rust-lang/cargo/blob/000d279306bf30f7c6fc482a89978a51ba10d8e5/src/cargo/util/rustc.rs#L62).